extensions/amp-story-auto-ads/0.1/test/test-story-ad-ui.js

```diff
@@ -216,4 +218,156 @@ describes.realWin('story-ad-ui', {amp: true}, (env) => {
       });
     });
   });
+
+  describe('createCta page outlink custom element', () => {
+    let buttonFitter;
+
+    beforeEach(() => {
```

Comment:
this is the same for all the new tests, can we move them to a single `describe` block?